src/controllers/activegate/capability/config.go

```diff
@@ -3,6 +3,18 @@ package capability
 const (
 	ActiveGateContainerName = "activegate"
 
+	ActiveGateVolumeNameGatewayConfig = "ag-lib-gateway-config"
```

Comment:
plz change consts names to pattern:
`ActiveGateVolumeNameGatewayConfig`-> `ActiveGateGatewayConfigVolumeName`

Reply:
we agreed to leave it as it is